src/info.plist

```diff
@@ -216,7 +216,7 @@ echo "DNS cache flushed"</string>
 				<key>escaping</key>
 				<integer>63</integer>
 				<key>script</key>
-				<string>ping -c 1 -t 6 {query} &amp;&gt; /dev/null &amp;&amp; echo "{query} is online." || echo "{query} is offline."</string>
+				<string>ping -c 1 -t 6 {query} &amp;&gt; /dev/null &amp;&amp; echo "{query} is online at $(command ping -c1 {query} | sed -nE 's/^PING[^(]+\(([^)]+)\).*/\1/p')." || echo "{query} is offline."</string>
```

Comment:
Why is `command` needed in this line? Wouldn't it just work without it too?

Reply:
It is a very rare use case, but some shell integrations (i.e _ZSH plugins_) override the default command for `ping` with a function. In my case, `ping` is actually a function that accepts and parses URLs in addition to domains (e.g. `ping https://github.com`). However, the fallacy would actually be on the plugin in this case, and it is unlikely that this would be loaded into the bash shell within Alfred anyways, so I'd say it's safe to take out!